packages/react-core/src/components/Wizard/examples/Wizard.md

```diff
@@ -3,18 +3,55 @@ id: Wizard
 section: components
 cssPrefix: pf-c-wizard
 propComponents:
-  ['Wizard', 'WizardNav', 'WizardNavItem', 'WizardHeader', 'WizardBody', 'WizardFooter', 'WizardToggle', 'WizardStep']
-ouia: true
+  [
+    'Wizard',
+    'WizardFooter',
+    'WizardFooterWrapper',
+    'WizardToggle',
+    'WizardStep',
+    'WizardBody',
+    'WizardHeader',
+    'WizardNav',
+    'WizardNavItem',
+    'WizardContextProps',
+    'WizardBasicStep',
+    'WizardParentStep',
+    'WizardSubStep',
+  ]
+beta: true
 ---
 
-import { Button, Drawer, DrawerActions, DrawerCloseButton, DrawerColorVariant,
-DrawerContent, DrawerContentBody, DrawerHead, DrawerPanelContent, DrawerSection, Wizard, WizardFooter, WizardContextConsumer, ModalVariant, Alert, EmptyState, EmptyStateHeader, EmptyStateFooter, EmptyStateBody, EmptyStateActions, Title, Progress, Form, FormGroup, TextInput } from '@patternfly/react-core';
+import {
+FormGroup,
+TextInput,
+Drawer,
+DrawerContent,
+Button,
+Title,
+Flex,
+Checkbox,
+DrawerPanelContent,
+DrawerColorVariant,
+DrawerHead,
+DrawerActions,
+DrawerCloseButton,
+Wizard,
+WizardFooter,
+WizardFooterWrapper,
+WizardToggle,
+WizardStep,
+WizardBody,
+useWizardFooter,
+useWizardContext,
+WizardNavItem,
+WizardNav,
+WizardHeader
+} from '@patternfly/react-core';
 import ExternalLinkAltIcon from '@patternfly/react-icons/dist/esm/icons/external-link-alt-icon';
 import SlackHashIcon from '@patternfly/react-icons/dist/esm/icons/slack-hash-icon';
 import CogsIcon from '@patternfly/react-icons/dist/esm/icons/cogs-icon';
 
-
-If you seek a wizard solution that allows for more composition, see the [React next](/components/wizard/react-next) tab.
+PatternFly has two implementations of a `Wizard`. This newer `Wizard` takes a more explicit and declarative approach compared to the older implementation, which can be found under the [React deprecated](/components/wizard/react-deprecated) tab.
```

Comment:
I think we can remove this blurb all together.  To me it can come across as you can use other, when in fact we want to  encourage use of the new Wizard.